Dockerfile.redhat

```diff
@@ -416,24 +416,24 @@ ENV PYTHONPATH=/ovms/lib/python:/ovms/python_deps
 
 WORKDIR /
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
-COPY ./install_redhat_gpu_drivers.sh ./install_gpu_drivers.sh
+COPY ./install_redhat_gpu_drivers.sh /install_gpu_drivers.sh
 # hadolint ignore=DL3003,DL3041,SC2164,SC1091
 RUN if [ -f /usr/bin/dnf ] ; then export DNF_TOOL=dnf ; echo -e "max_parallel_downloads=8\nretries=50" >> /etc/dnf/dnf.conf ; else export DNF_TOOL=microdnf ; fi ; \
     $DNF_TOOL upgrade --setopt=install_weak_deps=0 --nodocs -y ; \
     if [ "$GPU" == "1" ] ; then \
-    	source install_gpu_drivers.sh && rm -rf ./install_gpu_drivers.sh; \
+        source /install_gpu_drivers.sh && rm -rf /install_gpu_drivers.sh; \
     fi ; \
     # For image with Python enabled install Python library
     if ! [[ $debug_bazel_flags == *"py_off"* ]]; then \
         $DNF_TOOL install -y python39-libs --setopt=install_weak_deps=0 --nodocs; \
     fi ; \
-    $DNF_TOOL install -y shadow-utils; \
```

Comment:
shadow-utils is needed to allow adding new account in ubi-minimal base image. It is used when building without cuda to get minimal image size.